voorhoede/tudelft-spacefinder
# Display darker icons for the occupancy in the filter menu

# Changes

Replaces the default occupancy icons in the filter menu with darker versions. This way the icons are much clearer when they are placed on a grey background.

Before:
<img width="312" alt="Scherm­afbeelding 2023-05-17 om 17 45 36" src="https://github.com/voorhoede/tudelft-spacefinder/assets/3482946/ffcd8dd6-db4c-4a2e-8aeb-4685d110f811">

After:
<img width="315" alt="Scherm­afbeelding 2023-05-17 om 17 57 49" src="https://github.com/voorhoede/tudelft-spacefinder/assets/3482946/168f0a13-92ac-4b6d-9362-44e6e77010d7">

# Associated issue

Resolves https://trello.com/c/mu1DatLP/104-icons-in-filters-dont-look-aesthetically-pleasing

# How to test

1. Open preview link.
2. Open the filter menu and note that the new icons are visible.
3. The old icon should still be visible in the building cards, space cards, building pages, etc.

# Checklist
